<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>489</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="target_comboBox">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>210</y>
      <width>72</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="source_label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Source</string>
    </property>
   </widget>
   <widget class="QLabel" name="target_label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Target</string>
    </property>
   </widget>
   <widget class="QPushButton" name="translate_pushButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>390</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Translate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rotate_pushButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>180</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="source_comboBox">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>72</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="copy_pushButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>250</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="source_lineEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>381</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="target_lineEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>250</y>
      <width>381</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>489</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>source_lineEdit</tabstop>
  <tabstop>translate_pushButton</tabstop>
  <tabstop>rotate_pushButton</tabstop>
  <tabstop>source_comboBox</tabstop>
  <tabstop>copy_pushButton</tabstop>
  <tabstop>target_comboBox</tabstop>
  <tabstop>target_lineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
